server/zanata-frontend/src/app/styles/legacy.less

```diff
@@ -134,4 +134,11 @@
     word-wrap: break-word;
     white-space: pre-wrap;
   }
+ //fixes to Versions page
+ form#j_idt38.l--push-bottom-0 ~ .g.u-fullWidth {
+     height: 100vh;
+  }
+  #versions_form ~ ul.dropdown__content li {
```

Comment:
I think this should be outdented.

Reply:
It needs to be indented correctly (which I didn't do here, just fixed it) so that is is part of the wrapper `.new-zanata`. This ensures everything in this legacy file applies only to the jsf pages which have that top-level class.

And I will make sure to run the functional tests before merge.